autoscheduler/scraper/management/commands/scrape_courses.py

```diff
@@ -329,6 +326,27 @@ def save_models(instructors: List[Instructor], sections: List[Section], # pylint
 
     print(f"Saved all in {elapsed_time:.2f} seconds")
 
+def save_terms(terms, options):
+    """ Creates terms objects to save """
+
+    start = time.time()
+    now = timezone.now() # use timezone.now() so Django doesn't complain about naive times
+    with transaction.atomic():
+        # Bulk upsert
+        term_models = [Term(code=term, last_updated=now) for term in terms]
+
+        if options['year'] or options['recent']:
+            queryset = Term.objects.filter(code__in=terms)
+        else: # If no options are provided, then we're scraping all terms
+            queryset = Term.objects.all()
+
+        queryset.delete()
```

Comment:
This deletes all term models other than the ones you're scraping if you use the `-t` flag

Reply:
Great catch - man this is why we need `scrape_courses` tests